src/components/Layout/TooltipContent.js

```diff
@@ -85,7 +85,10 @@ export const TooltipContent = ({
         return itemDisplayNames
     }
 
-    const renderItems = (itemDisplayNames) => {
+    const renderItems = (itemDisplayNames = []) => {
+        if (itemDisplayNames.every((name) => name)) {
```

Comment:
```suggestion
        if (!itemDisplayNames.every((name) => name)) {
```

Reply:
I actually don't want this because if even one of the items is missing the name, then we don't want to show any of them.

Reply:
yeah so "all need to have a name", right. So that means "some dont have name" or "not every have name"... now it says "every has name", which is the inverse of what you want, because I was a bit too fast on the keyboard and forgot the "!", whoopsi..